src/sdg/core/management/utils.py

```diff
@@ -57,30 +57,36 @@ def load_informatiegebieden(data: List[Dict[str, Any]]) -> int:
     codes = [obj.pop("SDG_Code") for obj in data]
     grouped_data = dict(zip(codes, data))
 
-    informatiegebied_list = [
-        Informatiegebied(
+    count_informatiegebieden = 0
```

Comment:
This  variable is unused. Probably better to either remove it or add it to the returned count.